### PR TITLE
Apply voting UI review fixes

### PR DESCRIPTION
**Pull request by czarcas7ic**
Applies the voting UI follow-up fixes from the stacked review branch onto the upstream voting UI branch. This keeps voting flows account scoped during navigation, hardens submission edge cases, normalizes voting UI copy, and refreshes pending tally results instead of caching not-ready responses.

Tests:
- fvm flutter analyze --no-pub
- fvm flutter test --no-pub test/features/voting

### CODE REVIEW

**Comment by czarcas7ic** [comment]
@codex review

File: lib/src/features/voting/screens/voting_results_screen.dart [hunk 1]
```diff
@@ -59,10 +88,18 @@ class VotingResultsScreen extends ConsumerWidget {
               child: tally.when(
                 skipLoadingOnRefresh: false,
                 loading: () => const Center(child: CircularProgressIndicator()),
-                error: (error, _) => _Message("Couldn't load results: $error"),
+                error: (error, _) {
+                  final round = session.value?.round;
+                  if (round != null && _roundIsTallying(round)) {
+                    return _pendingResults();
```
[the remark this answers is not in the document]
Comment (czarcas7ic) [reply lib/src/features/voting/screens/voting_results_screen.dart]: https://github.com/chainapsis/vizor-wallet/pull/155/commits/a15848e3758bc52d11802f244a51ab15b0a3ed04

**Comment by czarcas7ic** [comment]
@codex review
